src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -622,4 +642,68 @@ public static void updateConfigErrorMessage(Config result, String key, String ms
       }
     }
   }
+
+  public static Map<String, String> getSchemaFromSchemaRegistry(
```

Comment:
nit: Javadocs for public methods 
(We follow some of effective java guidelines.
https://snowflakecomputing.atlassian.net/wiki/spaces/EN/pages/1960706106/Effective+Java+Guidelines+at+Snowflake)